client/blocks/comments/post-comment-list.jsx

```diff
@@ -81,16 +81,20 @@ class PostCommentList extends React.Component {
 	 * @param {object} props - the propes to use when evaluating if window should be scrolled down to a comment.
 	 * @returns {boolean} - whether or not we should scroll to a comment
 	 */
-	shouldScrollToComment = ( props = this.props ) =>
-		props.startingCommentId &&
-		props.commentsTree[ this.props.startingCommentId ] &&
-		props.commentsFetchingStatus.hasReceivedBefore &&
-		props.commentsFetchingStatus.hasReceivedAfter &&
-		! this.hasScrolledToComment &&
-		window.document.getElementById( `comment-${ props.startingCommentId }` );
+	shouldScrollToComment = ( props = this.props ) => {
+		return (
```

Comment:
hmmmm should we force this to a bool? right now it returns the dom node for success.